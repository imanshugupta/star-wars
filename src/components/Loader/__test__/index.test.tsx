import { render } from "@testing-library/react";
import { Loader } from "..";

describe("<Loader />", () => {
  test("match the snapshot", () => {
    const { asFragment } = render(<Loader />);
    expect(asFragment()).toMatchSnapshot();
  });
});
