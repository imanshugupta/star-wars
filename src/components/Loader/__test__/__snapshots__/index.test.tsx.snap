// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="loader"
  >
    <div
      class="spinner-border-sm spinner-border"
      role="status"
    >
      <span
        class="visually-hidden"
      >
        Loading...
      </span>
    </div>
  </div>
</DocumentFragment>
`;
